pkg/apis/apis/v1alpha1/types_apiexport.go

```diff
@@ -282,6 +282,8 @@ type APIExportStatus struct {
 	Conditions conditionsv1alpha1.Conditions `json:"conditions,omitempty"`
 
 	// virtualWorkspaces contains all APIExport virtual workspace URLs.
+	// This field will get deprecated in favor of APIExportEndpointSlice.
```

Comment:
that was my first intention but @stevekuznetsov found it a bit early, which I do agree. This could get deprecated when #2332 has been implemented and we have a working alternative to offer.